### PR TITLE
Deprecate Old SANS GUI

### LINKED ISSUE #24854: Deprecate old SANS GUI
The SANS instrument scientists have agreed that for the 3.14 release we can deprecate the old ISIS SANS GUI. This will be achieved by renaming the interfaces rather than posting a warning to the logger etc.
ISIS SANS -> ISIS SANS (deprecated)
ISIS SANS v2 experimental -> ISIS SANS

### PR DESCRIPTION
**Pull request by TTitcombe**
**Description of work.**
This PR changes the name of the two ISIS SANS GUIs. The old GUI is now `ISIS SANS (Deprecated)` and the "new" GUI is `ISIS SANS`.

**Report to:** nobody

**To test:**
1. MantidPlot -> Interfaces -> SANS. There should be, among others, `ISIS SANS (Deprecated)` and `ISIS SANS`. Check that both interfaces open
2. Workbench -> Interfaces -> SANS. There should be an `ISIS SANS` interface, but no deprecated one. Check that it opens.

Fixes #24854 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by Matthew-Andrew — DISMISSED** [review]
All works for me, good job on getting to this stage :+1:

**Comment by TTitcombe** [comment]
@Mantid-Matthew sorry it had a release notes conflict :cry:

**Review by Matthew-Andrew — DISMISSED** [review]

**Comment by SimonHeybrock** [comment]
There is a merge conflict now.

**Comment by NickDraper** [comment]
@TTitcombe 
Conflicts - A Haiku

```
To resolve conflicts
developer must merge
in the master branch
```

**Review by martyngigg — DISMISSED** [review]

File: docs/source/release/v4.0.0/sans.rst [hunk 1]
```diff
@@ -35,7 +35,6 @@ Improved
 * Added transmission sample/can to unsubtracted sample/can XML and H5 files when "Save Can" option selected.
 * Can set PhiMin, PhiMax, and UseMirror mask options from the options column in the table
 * Autocomplete for Sample shape column in the table.
-* Can separate items in variable q binning with commas or spaces. E.g. L/Q 0.0, 0.02 0.3 0.05, 0.8
```
Comment (DanNixon) [inline docs/source/release/v4.0.0/sans.rst]: Was this supposed to be removed?

Comment (TTitcombe) [reply docs/source/release/v4.0.0/sans.rst]: Yep it's a duplicate release note

File: docs/source/interfaces/ISIS SANS v2.rst [hunk 2]
```diff
@@ -14,9 +14,11 @@ Interface Overview
 ------------------
 
 This interface is used to reduce ISIS SANS data for SANS2D, LOQ,LARMOR and ZOOM.
-The interface can be accessed from the main menu of MantidPlot, in *Interfaces → SANS → ISIS SANS v2 experimental*.
+The interface can be accessed from the main menu of MantidPlot, in *Interfaces → SANS → ISIS SANS*.
 This interface is intended as a gradual replacement for the old ISIS SANS
-interface. Note that it is not yet feature complete and subject to change.
+interface. As of version 3.14 this interface was renamed from *ISIS SANS v2 experimental* to *ISIS SANS*, and the
```
Comment (DanNixon) [inline docs/source/interfaces/ISIS SANS v2.rst]: Should this be "version 4.0"?

Comment (TTitcombe) [reply docs/source/interfaces/ISIS SANS v2.rst]: Yep, that's been changed now

**Review by DanNixon — DISMISSED** [review]

**Review by martyngigg — APPROVED** [review]

**Comment by martyngigg** [comment]
Merging as Dan's previous comments have been addressed.

**Pull request merged**